### PR TITLE
Top langs include forks (First implements that forks count)

### PR DESCRIPTION
**Pull request by Bas950**
As requseted in https://github.com/anuraghazra/github-readme-stats/pull/480
Making different PRs for the new features

Made it this way since you wanted an opt-out instead of opt-in.

/api/top-langs/?username=USER
image
/api/top-langs/?username=USER&exclude_forks=true
image

Will make PRs for the other features after this one is merged, otherwise we will get merge conflicts.

### CODE REVIEW

**Comment by anuraghazra** [comment]
Okay LGTM, will merge later after adding tests and reviewing it.

**Comment by Bas950** [comment]
Changing to include_forks

**Comment by Bas950** [comment]
Updated to include_forks as requested in VC with @anuraghazra

**Comment by anuraghazra** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs. Thank you for your contributions.

Are you mad bro?

File: src/fetchers/top-languages-fetcher.js [hunk 1]
```diff
@@ -47,6 +53,13 @@ async function fetchTopLanguages(username, langsCount = 5, exclude_repo = []) {
   }
 
   let repoNodes = res.data.data.user.repositories.nodes;
+
+  if (!include_forks) {
+    repoNodes = repoNodes.filter((isFork) => {
+      return !isFork.isFork;
+    });
+  }
```
Comment (ensia96) [inline src/fetchers/top-languages-fetcher.js:61]:
```suggestion
  if (!include_forks {
    repoNodes = repoNodes.filter(({ isFork }) => !isFork);
  }
```

I think desctructuring would simplify filter callback : )

Comment (Bas950) [reply src/fetchers/top-languages-fetcher.js:61]: Will fix in the TS version after https://github.com/anuraghazra/github-readme-stats/pull/832 is merged.

Comment (ensia96) [reply src/fetchers/top-languages-fetcher.js:61]: Great! :)

**Comment by pplmx** [comment]
Any progress?

**Pull request closed without merging**